test/cases/regression-composer-works-behind-satellite.sh

```diff
@@ -24,22 +24,104 @@ function generate_certificates {
     sudo openssl x509 -req -in client.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out client.crt -days 365 -sha256
 }
 
+function cleanup {
+    greenprint "Putting things back to their previous configuration"
+    if [ -n "${REDHAT_REPO}" ] && [ "${REDHAT_REPO_BACKUP}" ];
+    then
+        lsattr "${REDHAT_REPO}"
+        chattr -i "${REDHAT_REPO}"
+        sudo rm -f "${REDHAT_REPO}"
+        sudo mv "${REDHAT_REPO_BACKUP}" "${REDHAT_REPO}" || echo "no redhat.repo"
+        sudo mv "${REPOSITORY_OVERRIDE}.backup" "${REPOSITORY_OVERRIDE}"
+    fi
+    if [[ -d /etc/httpd/conf.d.backup ]];
+    then
+        sudo rm -rf /etc/httpd/conf.d
+        sudo mv /etc/httpd/conf.d.backup /etc/httpd/conf.d
+    fi
+    sudo rm -f /etc/httpd/conf.d/repo1.conf
+    sudo rm -f /etc/httpd/conf.d/repo2.conf
+    sudo systemctl stop httpd || echo "failed to stop httpd"
+}
+
+trap cleanup EXIT
+
+# Provision the software under tet.
+/usr/libexec/osbuild-composer-test/provision.sh
+
+source /etc/os-release
 ARCH=$(uname -m)
-if [ "${ARCH}" = "x86_64" ];
-then
-    SNAPSHOT="20210326"
-elif [ "${ARCH}" = "aarch64" ];
+
+# Discover what system is installed on the runner
+case "${ID}" in
+    "fedora")
+        echo "Running on Fedora"
+        DISTRO_NAME="${ID}-${VERSION_ID}"
+        REPOSITORY_OVERRIDE="/etc/osbuild-composer/repositories/${ID}-${VERSION_ID}.json"
+        REPO1_NAME="fedora"
+        REPO2_NAME="updates"
+        ;;
+    "rhel")
+        echo "Running on RHEL"
+        case "${VERSION_ID}" in
```

Comment:
I suggest comparing only the major version against `${VERSION_ID%.*}`.

Reply:
Fixed, thanks 👍